### DISCUSSION TITLE
Paste Issue via context menu in theia Monaco

### DISCUSSION THREAD

**Discussion by ezhilkarthi** [eclipse-theia/theia#10640] [General]
Hi,
while clicking the paste in context menu inside the Theia Monaco I'm getting the below error.
![image](https://user-images.githubusercontent.com/31158252/149718905-b5e60c10-04c4-487d-ab50-21fa027b8c58.png)

**Comment by msujew** [comment]
Hi @ezhilkarthi,

I assume you are using Chrome to access Theia?

There is a bug regarding the support of the `paste` browser command on Chrome, which is why it is always disabled on Chrome:

https://github.com/eclipse-theia/theia/blob/0387822ed12e21fcb935c7af2c05ed294b782218/packages/core/src/browser/common-frontend-contribution.ts#L310-L315

However, the [associated bug ticket](https://bugs.chromium.org/p/chromium/issues/detail?id=757140) in the Chromium bug tracker seems to have been recently closed, so I guess it should be fine to depend on it now? @vince-fugnitto your opinion on that?

**Reply by vince-fugnitto** [reply]
@msujew I'd be fine with fixing the behavior if it is fixed in chrome :+1: I'm not sure if we should try to detect a version of chrome (that has the fix) however.

**Reply by msujew** [reply]
It seems like the fix was actually introduced in a [commit from 2017](https://bugs.chromium.org/p/chromium/issues/detail?id=757140#c9), which wasn't correctly picked up by the bug tracker. So I assume we can safely assume that most users of Theia will be using a Chrome version without the buggy implementation.

**Reply by msujew** [reply]
I created an issue for this: #10642. @ezhilkarthi Thanks for the heads-up.

**Reply by ezhilkarthi** [reply]
Hi,
I'm facing the same issue in firefox also.
